src/Nethermind/Nethermind.TxPool/TxPool.cs

```diff
@@ -191,10 +194,6 @@ private void OnHeadChange(object? sender, BlockReplacementEventArgs e)
         {
             try
             {
-                // Clear snapshot
-                _transactionSnapshot = null;
-                _blobTransactionSnapshot = null;
-                _hashCache.ClearCurrentBlockCache();
```

Comment:
Could that lead to some inconsistencies in TxPool, so it could give us stale data in normal (not archive) processing mode?

Reply:
Shouldn't do as we do the bucket Update etc on the other half where this moves to